modules/nf-core/lai/environment.yml

```diff
@@ -0,0 +1,9 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/environment-schema.json
+name: "lai"
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - "bioconda::LTR_retriever=2.9.0"
```

Comment:
Yes, I agree. Let's move it to `LTRRETRIEVER_LAI` because historically LAI never had its own GitHub repo or a Bioconda recipe. It is a script which has its own publication but can only be applied to the outputs from `LTRRETRIEVER`.

Reply:
Now the module is `LTRRETRIEVER_LAI`